scripts/run-e2e-test.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/sh
+
+set -o errexit
+set -o pipefail
```

Comment:
`pipefail` (at least) is a bash-ism so you need `#! /bin/bash` shebang. Not sure about `errexit` or `nounset`, the short names (`-e` and `-u`) are portable I think.

Given how awful it is to replicate `pipefail` in pure POSIX shell I think just using `/bin/bash` for the shebang is the right answer in this case.

Reply:
Ah, I didn't realise this... It's okay, we aren't relying on `pipefail` here. I'll switch to `/bin/sh` and use `set -eu`, same goes for `pkg/kube-e2e-test/e2e.sh` (which currently has an explicit `set +o pipefail`).